### DISCUSSION TITLE
proofreader / playtester sought

### DISCUSSION THREAD

**Discussion by cpboyko** [tacosontitan/tacosontitan#10] [Show and tell]
Hi Taco!

I found this discussions page on your profile at puzzling.stackexchange, where about a year ago I posted a few puzzles from the novel I've been working on. You were kind enough then to offer a brilliant solution or two. Thanks!

I've now finished a draft of the "electronic escape novel", and am looking for puzzlers who know a bit of coding to take it for a test drive. 

I can't offer more than a nominal honorarium and/or an acknowledgement, but from your activity on puzzles.se I thought this might be so up your alley that you'd enjoy tackling it for fun.

Here's the Craigslist ad that I posted, and that got zero replies!:
"Seeking proofreaders / playtesters of electronic escape novel. Love, or at least rudimentary knowledge, of puzzles, programming, and poems an asset. Pays a nominal honorarium, calculated as 1000 ^ (4n/267 - 2/3) CAD (Canadian dollars), where n is the number of chapters (of 89 total) that you successfully decipher."

Warm best wishes,
Craig (C. P. Boyko)

**Comment by tacosontitan** [comment]
@cpboyko, there's literally nothing to go on here and I will **not** navigate to external links. You can feel free to share any material you'd like some feedback on here.

**Comment by cpboyko** [comment]
I see. Perhaps we're at an impasse: I don't want to put the book online.

If you're willing to take a chance that I'm not a spammer, shoot me a collegial nod at obnakwa [at] symux [dot] oay

xo O. B.

**Reply by tacosontitan** [reply]
![TheEmperorsNewGrooveKuzcoGIF](https://github.com/tacosontitan/tacosontitan/assets/65432314/3a7ecb27-ee43-46a9-b568-c5256b4b4b5d)
